src/BinSkim.Driver.FunctionalTests/BaselineTestsData/NonWindowsExpected/gcc.stack_protector.a.sarif

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "id": "ab35e855-7b20-4de6-9372-d94bef6e9c91",
+      "tool": {
+        "name": "BinSkim",
+        "fullName": "BinSkim 1.5.0.0",
+        "version": "1.5.0.0",
+        "semanticVersion": "1.5.0",
+        "sarifLoggerVersion": "1.7.3.0",
+        "language": "en-US",
+        "properties": {
+          "Comments": "A security and correctness analyzer for portable executable and MSIL formats.",
+          "CompanyName": "Microsoft",
+          "ProductName": "BinSkim Portable Executable Analyzer"
+        }
+      },
+      "results": [],
+      "configurationNotifications": [
+        {
+          "id": "WRN998.NotSupportedPlatform",
+          "physicalLocation": {},
+          "message": "Rule 'BuildWithSecureTools' was disabled as it cannot run on the current platform 'Windows'.  It can only run on 'Linux'.",
```

Comment:
Message is incorrect here due to https://github.com/Microsoft/sarif-sdk/pull/767